src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -4046,7 +4078,16 @@ public Guid ParentDebuggerId
 
         #endregion
 
-        #region Overrides
+        #region Overridesbp
+
+        /// <summary>
+        /// Adds the provided set of breakpoints to the debugger.
+        /// </summary>
+        /// <param name="breakpoints">Breakpoints.</param>
+        public override void SetBreakpoints(IEnumerable<Breakpoint> breakpoints)
```

Comment:
Style: => _wrappedDebugger.SetBreakpoints(breakpoints);

Reply:
This is how the code was originally but I can change it